templates/cbs-tools/scripts/create_sig.sh

```diff
@@ -221,6 +221,10 @@ do
                     fi
                     if [[ "x$DIST" == "x8" ||  "x$DIST" == "x8s" ]]
                     then
+                        if ( $HA_REPO_ENABLED )
```

Comment:
HA_REPO_ENABLED has no default ? I see you declare it only in the openstack config, but should then be declared as "false" in default config ? 
Probably in this line for common : https://github.com/CentOS/ansible-role-kojihub/blob/master/templates/cbs-tools/scripts/create_sig.sh#L262 ?